### DISCUSSION TITLE
Combobox with multiline options

### DISCUSSION THREAD

**Discussion by kumar-tadepalli** [vmware-clarity/ng-clarity#636] [Q&A] [answered]
Hi,

Is it possible to have option with multilines and custom template in combobox?

Something like this:
![image](https://user-images.githubusercontent.com/12848838/231210820-5794e921-d7a7-4cd3-8313-e211fbf362af.png)

**Comment by kumar-tadepalli** [comment]
Hi,
Can someone from Clarity team confirm on this?

**Reply by kevinbuhmann** [reply, marked as answer]
Yes, I believe you can implement this design by projecting such content into the `clr-option` element.

Example: https://stackblitz.com/edit/clarity-light-theme-clr15-hwdfbt?file=src%2Fapp%2Fapp.component.html

Note that the combobox can only filter based on the items in the collection iterated by `*clrOptionItems`, not the full content of the `clr-option` element.

**Reply by kumar-tadepalli** [reply]
@kevinbuhmann  I tried the same approach with select component.
It is not working. Is it possible for select?

**Reply by kevinbuhmann** [reply]
The `select` element is a native HTML element, not a Clarity component. I don't think multiline options in a `select` are possible.

**Reply by kumart-netapp** [reply]
There is no way I can have id binded to option value and show the name as text in options and selection text.
It will always be complex object in real time and options value should be binded to id instead of whole object to support edit cases.
